packages/client/hmi-client/src/services/models/funman-service.ts

```diff
@@ -135,22 +135,22 @@ interface FunmanBoundingBox {
 	x2: number;
 	y2: number;
 }
-export const getBoxes = (
-	processedData: FunmanProcessedData,
-	param1: string,
-	param2: string,
-	_timestep: number,
-	boxType: string
-) => {
+export const getBoxes = (processedData: FunmanProcessedData, param1: string, param2: string, boxType: string) => {
 	const result: FunmanBoundingBox[] = [];
 
 	const temp = processedData.boxes
```

Comment:
May want to leave a comment 141 -> 147 explaining that we are grabbing all ub timesteps to get the latest ub timestep for later use in ln 154
however it is readable so not required